pycbc/pnutils.py

```diff
@@ -30,33 +30,36 @@
 import lal, lalsimulation
 import numpy
 from scipy.optimize import bisect, brentq, minimize
+from pycbc import derived_parameters
```

Comment:
Yeah, I didn't want to go searching through pycbc and replace all instances of pnutils that call these functions at the moment. Sure, I'll open an issue after this patch is pushed.